__fixtures__/test-project/api/src/services/contacts/contacts.test.ts

```diff
@@ -37,7 +39,7 @@ describe('contacts', () => {
   })
 
   scenario('updates a contact', async (scenario: StandardScenario) => {
-    const original = await contact({ id: scenario.contact.one.id })
+    const original = (await contact({ id: scenario.contact.one.id })) as Contact
```

Comment:
You’re probably right - My focus wasn’t so much on assessing what the generators generate, more on satisfying TS. The type casting here is fine though, because the test will fail if contact is null. This is what discussed on the call a few weeks ago - but happy to try a different way, but am not aware of the original reasoning.

Reply:
I just realised, because Id Is auto generated by prisma, so we don’t know the Id ahead of time.

Reply:
> The type casting here is fine though, because the test will fail if contact is null. This is what discussed on the call a few weeks ago
Thanks for jogging my memory, I remember that now 👍